### PR TITLE
Merge Back 0.15.0

### PR DESCRIPTION
**Pull request by micbar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## [0.15.0] - 2019-12-04
+
+### Fixed
+
+- Don't delete / disable Users if the Change the DN - [#470](https://github.com/owncloud/user_ldap/issues/470)
```
Comment (phil-davis) [inline CHANGELOG.md]: `if the Change the DN` - this does not make sense to me, I have to try and guess what it might mean.

Comment (micbar) [reply CHANGELOG.md]: Use Case: A LDAP user is moved in AD Server because of changing a department

Comment (phil-davis) [inline CHANGELOG.md]:
```suggestion
- Don't delete / disable Users if they change their DN - [#470](https://github.com/owncloud/user_ldap/issues/470)
```
Maybe this is the meaning?

**Comment by HanaGemela** [comment]
Closes https://github.com/owncloud/user_ldap/issues/481

**Comment by micbar** [comment]
@PVince81 Did something change in CI?
Build is failing now.

**Comment by phil-davis** [comment]
Assigned myself so I see this when I am back at a real computer

**Comment by phil-davis** [comment]
New acceptance test scenarios for new features got merged in core late last week. I suspect that the new tests got executed, but the core `daily-master-qa` tarball had not yet got updated with the feature code. I restarted CI just now, let's see.

**Comment by phil-davis** [comment]
@micbar @HanaGemela this is passing now, and 0.15.0 release has been tagged... 
So IMO this can be merged.

**Pull request merged**